windows/CodePush.Net46/FileUtils.cs

```diff
@@ -0,0 +1,62 @@
+﻿using PCLStorage;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace CodePush.ReactNative
+{
+    internal class FileUtils
+    {
+        internal async static Task MergeFoldersAsync(IFolder source, IFolder target)
+        {
+            foreach (IFile sourceFile in await source.GetFilesAsync().ConfigureAwait(false))
+            {
+                await CopyFileAsync(sourceFile.Path, Path.Combine(target.Path, sourceFile.Name)).ConfigureAwait(false);
+            }
+
+            foreach (IFolder sourceDirectory in await source.GetFoldersAsync().ConfigureAwait(false))
+            {
+                var nextTargetSubDir = await target.CreateFolderAsync(sourceDirectory.Name, CreationCollisionOption.OpenIfExists).ConfigureAwait(false);
+                await MergeFoldersAsync(sourceDirectory, nextTargetSubDir).ConfigureAwait(false);
+            }
+        }
+        internal async static Task ClearReactDevBundleCacheAsync()
+        {
+            try
+            {
+
+                var devBundleCacheFile = await FileSystem.Current.LocalStorage.GetFileAsync(CodePushConstants.ReactDevBundleCacheFileName).ConfigureAwait(false);
+
+                if (devBundleCacheFile != null)
+                {
+                    await devBundleCacheFile.DeleteAsync().ConfigureAwait(false);
+                }
+            }
+            catch (FileNotFoundException)
```

Comment:
>catch [](start = 12, length = 5)

Is there a "TryGetFileAsync" or the like to prevent throwing an exception in a common case?  Is this method typically called when the bundle does not exist?